src/lib/rest/StringFilter.cpp

```diff
@@ -42,6 +42,27 @@ using namespace mongo;
 
 
 
+// Temporary debug function
```

Comment:
To avoid "leaking" this function into final develop branch, please add a "FIXME PR" in this place.